tests/installation/bootloader_svirt.pm

```diff
@@ -236,8 +236,10 @@ sub run {
     if ($vmm_family eq 'kvm') {
         $iface_model = 'virtio';
     }
-    elsif ($vmm_family eq 'xen' && $vmm_type eq 'hvm') {
-        $iface_model = 'netfront';
+    elsif ($vmm_family eq 'xen') {
+        $ifacecfg{type}        = 'bridge';
+        $ifacecfg{source}      = {bridge => 'br0'};
+        $ifacecfg{virtualport} = {type   => 'openvswitch'};
     }
     elsif ($vmm_family eq 'vmware') {
```

Comment:
The only very minor comment I have, is that there should by a `else`, otherwise you might run into a undefined state for `$iface_model`. Let it be a simple `die`.

Reply:
I guess it is the result of separate bootloader test cases for `hyperV` and other `svirt` based hypervisors as `xens` and `vmware`. However, `tests/installation/bootloader_svirt.pm` is not used by `hyperv`, so it seems to be safe to `die` over here.
I agree that it's a good practise to include `else` statement, thanks!